frontend/src/components/AboutUs.jsx

```diff
@@ -0,0 +1,84 @@
+import "./AboutUs.scss";
+
+function AboutUs() {
+  return (
+    <div className="about-container">
+      <div className="text-area">
+        <h2 className="about-title">About Us</h2>
+        <p>
+          Welcome to Cocktail-Atom, your ultimate destination for discovering
+          the world of cocktails with a simple click! Founded by a dynamic trio
+          : Khachik Sahakyan, Quentin Vayssieres, and Tristan Delmas
+          <br />
+          <br />
+          Cocktail-Atom is the fruit of our collective passion for mixology and
+          the result of our journey through the Wild Code School's web and
+          mobile development program.
+        </p>
+
+        <h3 className="about-article">Our Mission</h3>
+        <p>
+          At Cocktail-Atom, we are on a mission to make cocktail exploration an
+          enjoyable and seamless experience for enthusiasts and casual drinkers
+          alike.
+          <br />
+          <br />
+          By leveraging the power of the CocktailDB API, we've curated an
+          extensive database that allows you to find the perfect concoction
+          based on your preferences. Whether you're a seasoned mixologist or a
+          curious novice, our platform is designed to elevate your cocktail
+          adventure.
+        </p>
+
+        <h3 className="about-article">The Team Behind the Shaker</h3>
+        <p>
+          <strong>Khachik Sahakyan:</strong> The driving force behind our
+          commitment to quality and continuous learning. Khachik is determined
+          to provide top-notch features while staying at the forefront of new
+          techniques. His passion for excellence sets the standard for
+          Cocktail-Atom's functionality.
+        </p>
+
+        <p>
+          <strong>Quentin Vayssieres:</strong> A maestro of aesthetic
+          functionality, Quentin's focus is on creating features that not only
+          work seamlessly but also make users feel right at home. His dedication
+          to user experience transforms Cocktail-Atom into an inviting space for
+          cocktail enthusiasts.
+        </p>
+
+        <p>
+          <strong>Tristan Delmas:</strong> With a focus on achieving optimal
```

Comment:
Seekings continuous improvement, always exploring ways to enhance the user interface and overall performance of Cocktail-Atom => Seeking continuous improvement, he's always exploring ways to enhance the user interface and overall performance of Cocktail-Atom.